dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Register/CheckAnswers.cshtml

```diff
@@ -56,7 +58,9 @@
                     {
                         <govuk-summary-list-row>
                             <govuk-summary-list-row-key>National Insurance number</govuk-summary-list-row-key>
-                            <govuk-summary-list-row-value>@(Model.NationalInsuranceNumber ?? "Not given")</govuk-summary-list-row-value>
+                            <govuk-summary-list-row-value>
+                                <span fallback-text="Not given">@Model.NationalInsuranceNumber</span>
```

Comment:
We should probably be consistent and choose one of 'Not given' and 'Not provided'